src/preset/indexer.js

```diff
@@ -1,8 +1,15 @@
-import { extractStories } from '../parser/extract-stories';
 import fs from 'fs-extra';
+import * as svelte from 'svelte/compiler';
+import { extractStories } from '../parser/extract-stories';
+import { loadSvelteConfig } from '../config-loader';
 
 export async function svelteIndexer(fileName, { makeTitle }) {
   let code = (await fs.readFile(fileName, 'utf-8')).toString();
+  const svelteOptions = loadSvelteConfig();
```

Comment:
This resolves a promise, which made it not work for me. When I added `await` here, it worked.

```suggestion
  const svelteOptions = await loadSvelteConfig();
```